designs/tripleo/osc-tripleo.md

```diff
@@ -73,6 +73,20 @@ description: >
     description: image
     type: string
 	default: "/var/lib/osc/data"
+		
+  # The parameter contains the path where OSC certificate file resides.
+  OSCTrustStoreFile:
+    default: '/osc-cert/osctruststore.jks' 
```

Comment:
This addresses how the user, during deployment, can communicate where the cert is to OSC. But how does the cert end up on this path?

Reply:
My assumption is that system administrator before deploying OSC service shall create or get the relevant certificates from CA and adds' this in director node (in the relevant directory) before initiating the deployment process. This is how i understood from ODL. It is a good point, I will add this assumption in the document so that it can be discussed in the upcoming review with other stakeholders.